daemon/prune.go

```diff
@@ -25,6 +25,27 @@ var (
 	// ErrPruneRunning is returned when a prune request is received while
 	// one is in progress
 	ErrPruneRunning = fmt.Errorf("a prune operation is already running")
+
+	containersAcceptedFilters = map[string]bool{
+		"label":  true,
+		"label!": true,
```

Comment:
Should the `!` actually be part of the filter name? Not sure how it's handled later on, but looks a bit tricky (i.e. `label!=foo.bar` would be accepted, but not `label != foo.bar`, correct?)